### DISCUSSION TITLE
Nodes can get lost

### DISCUSSION THREAD

**Discussion by ghost** [IndiePython/nodezator#86] [Concerns/constructive criticism]
I've read the reason for why zoom will not be implemented, totally understandable.

My problem is that I got an error for an unconnected node (I only know it's ID), but I've no idea where it resides in the map.

Sure, I can get rid of it in the file with VIM, but that's cumbersome.

Maybe a way to jump to an ID?

**Comment by KennedyRichard** [comment]
Totally, you are right. I'm thinking of introducing a key combination for quickly jumping to a node with a given id, just like you described. This seems crucial, specially for when dealing with large graphs.

I also want to introduce some kind of map view in the future, as [mentioned in the past](https://github.com/IndiePython/nodezator/discussions/14#discussioncomment-3057031) and depicted below:

![map view](https://camo.githubusercontent.com/45e87a740f0c6bf4c40d39ddb610135c9b4300e9da767a5b4807477e2c145249/68747470733a2f2f692e696d6775722e636f6d2f4953386636774e2e706e67)

A sort of overview of the entire graph or at least large parts of it. I'll reclassify this as a minor issue and deal with those things in the future/near future.

The "jump to node of given id" feature should probably land first, since it is more quick and straightforward to implement. The map view should take a bit more time, although it should be easy to implement as well. The extra time needed for the map view feature is due to the number of different services involved and the need for fine-tuning the solution.

Peace.
